src/Raven.Studio/typescript/viewmodels/database/tasks/importDatabase.ts

```diff
@@ -78,6 +78,10 @@ class importDatabase extends viewModelBase {
                     return;
                 }
 
+                if (percentComplete === 100) {
+                    setTimeout(() => this.isUploading(false), 700);
```

Comment:
Or actually I have one question to this PR:

Why do we have this timeout? Can't we just set it to false when upload is completed?